specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-04-01-preview/WorkspaceManagerAssignments.json

```diff
@@ -0,0 +1,736 @@
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "WorkspaceManagerAssignment": {
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/AzureEntityResource"
+        }
+      ],
+      "description": "The workspace manager assignment",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/WorkspaceManagerAssignmentProperties",
+          "description": "The workspace manager assignment object",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "WorkspaceManagerAssignmentProperties": {
+      "description": "The workspace manager assignment properties",
+      "properties": {
+        "targetResourceName": {
+          "description": "The name of the workspace manager group targeted by the workspace manager assignment",
+          "type": "string"
+        },
+        "lastJobEndTime": {
+          "description": "The time the last job associated to this assignment ended at",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastJobProvisioningState": {
+          "description": "State of the last job associated to this assignment",
+          "enum": [
+            "Succeeded",
+            "InProgress",
+            "Canceled",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "provisioningState",
+            "values": [
+              {
+                "description": "The job succeeded",
+                "value": "Succeeded"
+              },
+              {
+                "description": "The job was canceled",
+                "value": "Canceled"
+              },
+              {
+                "description": "The job is in progress",
+                "value": "InProgress"
+              },
+              {
+                "description": "The job failed",
+                "value": "Failed"
+              }
+            ]
+          }
+        },
+        "items": {
+          "description": "List of resources included in this workspace manager assignment",
+          "items": {
+            "$ref": "#/definitions/assignmentItem"
+          },
+          "x-ms-identifiers": [],
+          "type": "array"
+        }
+      },
+      "required": [
+        "targetResourceName",
+        "items"
+      ],
+      "type": "object"
+    },
+    "assignmentItem": {
```

Comment:
>assignmentItem [](http://example.com/codeflow?start=5&length=14)

aren't types/definitions supposed to be declared PascalCase? [if this is not going to break anyone]

Reply:
We have customers using the APIs currently